dts/msc-sm2-imx6x.dtsi

```diff
@@ -483,10 +483,10 @@
 			regulator-min-microvolt = <5000000>;
 			regulator-max-microvolt = <5000000>;
 			regulator-always-on;
-/* TODO: This pin will be first set to output-low and the set high. The glitch this produces is propagated onto hdmi_pwr, upsetting the touch screen controller.
```

Comment:
I don't understand what is fixed here, does the 
/*
*/ 
commenting scheme not work in device trees?

Reply:
It does. Both C comments (/* */) and C++ comments (// ) are allowed, but real men only the first for comments.

Reply:
Good argument lol